ros2nodl/ros2nodl/verb/validate.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it under the terms of the
+# GNU Limited General Public License version 3, as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+# without even the implied warranties of MERCHANTABILITY, SATISFACTORY QUALITY, or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Limited General Public License for more details.
+#
+# You should have received a copy of the GNU Limited General Public License along with
+# this program. If not, see <http://www.gnu.org/licenses/>.
+
+from pathlib import Path
+import sys
+from typing import TYPE_CHECKING
+
+from argcomplete.completers import FilesCompleter
+import nodl
+from ros2nodl.verb import VerbExtension
+
+if TYPE_CHECKING:  # pragma: no cover
+    import argparse
+
+
+class ValidateVerb(VerbExtension):
+    """Validate NoDL XML documents."""
+
+    def add_arguments(self, parser: 'argparse.ArgumentParser', *_):  # pragma: no cover
```

Comment:
Ditto previous comment about `*_`. Also, why the pragma here?

Reply:
Fixed in b8c715a